python/trezorlib/tests/device_tests/test_msg_signtx.py

```diff
@@ -840,3 +844,47 @@ def test_change_on_main_chain_allowed(self):
         )
 
         check_sign_tx(self.client, "Testnet", [inp1], [out1, out_change])
+
+    def test_tpos(self):
```

Comment:
This test fails because Stakenet doesn't have a blockbook which is required for retriving transactions, is there a way to avoid that? possibly storing a json with the transaction details to let the test succeed.

Reply:
Just put the TX in the json format in `tests/txcache`

Reply:
Done